### PR TITLE
refactor repo name and schema-ui usage

### PR DESCRIPTION
**Pull request by sanuann**

### CODE REVIEW

**Comment by sanuann** [comment]
@satra the schema-ui refactoring is done and context urls for activities point to reproschema-library. Can I merge this now?

File: .github/workflows/build_deploy.yml [hunk 1]
```diff
@@ -37,7 +37,9 @@ jobs:
     # Runs a set of commands using the runners shell
     - name: Run a multi-line script
       run: |
-        git submodule update --init --recursive
+        curl -O -sSL https://github.com/ReproNim/reproschema-ui/archive/8519d6141f68f277f1e36c1b1be62810bb1bdc93.zip
+        unzip 8519d6141f68f277f1e36c1b1be62810bb1bdc93.zip
```
Comment (satra) [inline .github/workflows/build_deploy.yml:41]: do you want to use this version of schema ui? or the current master checksum: https://github.com/ReproNim/reproschema-ui/tree/0ffbfe965e2931b93a3efc620b14551d29e5707a

Comment (sanuann) [reply .github/workflows/build_deploy.yml:41]: yes, the current master. I'll update it. how do you get that URL with the checksum?

File: ui-changes/config/index.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
-'use strict'
+'use strict';
```
Comment (satra) [inline ui-changes/config/index.js:1]: this file should be deleted.

File: activity/covid19/covid19_schema [hunk 3]
```diff
@@ -2,8 +2,9 @@
     "@context": [
         "https://raw.githubusercontent.com/ReproNim/reproschema/refactor-1.0/contexts/generic",
         {
-            "wellbeing": "https://raw.githubusercontent.com/ReproNim/reproschema/master/activities/Wellbeing/items/",
-            "nda": "https://raw.githubusercontent.com/ReproNim/reproschema/master/activities/NDA/items/"
+            "wellbeing": "https://raw.githubusercontent.com/ReproNim/reproschema-library/refactor-1.0/activities/Wellbeing/items/",
+            "nda": "https://raw.githubusercontent.com/ReproNim/reproschema-library/refactor-1.0/activities/NDA/items/",
+            "demo": "https://raw.githubusercontent.com/sanuann/reproschema-library/master/activities/demographics_and_background_information_v1/items/"
```
Comment (satra) [inline activity/covid19/covid19_schema:7]: should this point to your repo?

Comment (sanuann) [reply activity/covid19/covid19_schema]: it wasn't merged to repronim yet. changed it

**Pull request merged**